### DISCUSSION TITLE
Install directory

### DISCUSSION THREAD

**Discussion by lyingsackoffun** [neonbjb/tortoise-tts#248] [Q&A]
Can someone help me change it? Windows won't let me install to the default python directory so I gotta install tortoise on another one. I'm dying to try this on a few novels but Windows has thrown curve balls at me on EVERY. SINGLE. STEP.

**Comment by lyingsackoffun** [comment]
I've come to the conclusion that this just won't work on Windows. I had to reinstall python because it stopped working out of nowhere and can't get it set up again. Too many issues to solve and many issues were unsolvable entirely no matter how hard I tried. If it isn't a windows issue it's a python issue and vice versa. I'm giving up on this. Maybe someone can make this work without all the headache one day.

**Comment by neonbjb** [comment]
Hey, sorry you're having issues. Python is already a bit of a hot mess, and given that Windows is a second class citizen for it, it's even worse. For whatever it is worth, I actively built this on a Windows development PC and regularly use it even today on Windows. It's certainly possible, if frustrating.

**Reply by lyingsackoffun** [reply]
> Hey, sorry you're having issues. Python is already a bit of a hot mess, and given that Windows is a second class citizen for it, it's even worse. For whatever it is worth, I actively built this on a Windows development PC and regularly use it even today on Windows. It's certainly possible, if frustrating.

To say it's frustrating is an understatement but I've had bad luck with Windows from day 1. I only keep it for gaming reasons. As for the problems I'm having. I think it's corrupted files or something. Can't find a solution but but after the shenanigans I think it's a Windows problem. I'll fix it eventually.
